app/lib/actions/katello/upstream_subscriptions/bind_entitlements.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Katello
+    module UpstreamSubscriptions
+      class BindEntitlements < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :organization_id, Integer
+          param :pools, Array
+        end
+
+        def plan(organization_id = nil, pools = [])
+          plan_self(organization_id: organization_id, pools: pools)
+          return unless pools.any?
+          plan_action(Katello::Organization::ManifestRefresh, ::Organization.find(organization_id))
```

Comment:
You probably want these in a 'sequence do ... end' block.  Dynflow unfortunately defaults to 'concurrent' which means these two run phases get run together.